thunder-janelia

```diff
@@ -11,6 +11,7 @@ parser = argparse.ArgumentParser(description="setup and run thunder")
 parser.add_argument("task", choices=('install', 'update', 'start'), default='start')
 parser.add_argument("-v", "--sparkversion", type=str, default='1', required=False)
 parser.add_argument("-i", "--ipython", action="store_true")
+parser.add_argument("-p", "--thunderpath", type=str, default='~/thunder', required=False)
```

Comment:
I'd change the flag to `-t`, more consistent with using the single letter version as the first letter of the option.